echopype/tests/echodata/test_echodata.py

```diff
@@ -459,12 +459,16 @@ def test_nan_range_entries(range_check_files):
             echodata, env_params=None, cal_params=None, ecs_file=None, waveform_mode="BB", encode_mode="complex",
         )
         range_output = cal_obj.range_meter
-        nan_locs_backscatter_r = ~echodata["Sonar/Beam_group1"].backscatter_r.isel(beam=0).drop("beam").isnull()
+        if "beam" in echodata["Sonar/Beam_group1"].dims:
+            nan_locs_backscatter_r = ~echodata["Sonar/Beam_group1"].backscatter_r.isel(beam=0).drop(
+                "beam").isnull()
+        else:
+            nan_locs_backscatter_r = ~echodata["Sonar/Beam_group1"].backscatter_r.isnull()
```

Comment:
this is not needed as the file is EK80 broadband complex data -- the `beam` dimension always needs to be dropped.

Reply:
Done